package.xml

```diff
@@ -1,11 +1,16 @@
 <package>
   <name>srdfdom</name>
-  <version>0.2.7</version>
+  <version>0.2.8</version>
   <description>Parser for Semantic Robot Description Format (SRDF).</description>
   <author email="isucan@willowgarage.edu">Ioan Sucan</author>
+  <author email="gwalck@techfak.uni-bielefeld.de">Guillaume Walck</author>
+  <author>Wim Meeussen</author>
+  <author>John Hsu</author>
```

Comment:
It comes from the urdfdom from which I copied the parser and adapted it to srdf (https://raw.githubusercontent.com/ros-gbp/urdfdom-release/debian/indigo/trusty/urdfdom/package.xml) 
But if looking in details, the people added in the README.md are the correct ones I derived the work from. 
I don't know how to acknowledge these people.